backend/main.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3.10
+import asyncio
+import logging
+import sys
+from datetime import datetime
+from typing import List
+
+from aiohttp import WSMsgType, web
+
+from backend import schemas
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+websockets: List[web.WebSocketResponse] = []
+
+
+async def websocket_handler(request):
+    """Main WebSocket handler"""
+    ws = web.WebSocketResponse()
+    await ws.prepare(request)
+    websockets.append(ws)
+
+    async for msg in ws:
+        if msg.type == WSMsgType.TEXT:
+            message = schemas.BaseMessage.parse_raw(msg.data)
+            logger.debug(f"Received message: {message.dict()}")
+        elif msg.type == WSMsgType.ERROR:
+            logger.error(f'ws connection closed with exception {ws.exception()}')
```

Comment:
Minor comment: Do we want to have our strings in single quotes(line 29, 32) or double quotes(line 27, etc)? Should we try to establish an agreed-upon common style for this?

Reply:
I think that there isn't any specific rule to this, but we can use double quotes for all strings except "small symbol-symbol like strings" https://stackoverflow.com/a/56190